### DISCUSSION TITLE
Recebimentos esporádicos

### DISCUSSION THREAD

**Discussion by elMor3no** [DyegoCosta/trabalhando-remoto#168] [General]
Obrigado pelas informações, muito úteis.

Tenho uma dúvida sobre pagamentos recebidos esporadicamente do exterior:

1.- Para reduzir as taxas de transferência cobradas pelos bancos, costumo acumular 3 ou 4 meses para solicitar o pagamento. Neste caso a cobrança da Alíquota será fixada no valor maior (22,5 - 27,5)? 

2.- Como não os pagos só acontecem cada 3 ou 4 meses é viável ter um PJ o melhor continuar como autônomo?

3.- O pagamento do INSS é obrigatório? Deveria ser feito mensalmente ou apenas quando recebo os pagamentos?

4.- Como é feito o pagamento do INSS? Este é um ponto que ainda não está totalmente claro para mim.

**Comment by copercini** [comment]
1- Como pessoa física, vc paga a tabela do imposto de renda sobre o valor, mas como é anual, vc vai provavelmente pagar os 27,5% quando receber 3 meses de uma vez e nada nos meses que não receber nada, e então ter que declarar o imposto de renda para receber a restituição do que vc pagou a mais

2- Depende do valor, PJ tem custos fixos (contabilidade, certificado digital) que vc precisa pagar todos os meses, recebendo ou não. O ideal é fazer as contas certinho, PJ é mais burocrático, mas tem uma redução boa de impostos

3- Como PJ, nos meses que vc não recebe nada, geralmente se tira um salário mínimo de pro-labore e paga o INSS sobre isso, como PF não sei como funciona

4- Como PJ, a contabilidade faz o boleto. Como PF não sei como funciona
